src/display/api.js

```diff
@@ -1742,7 +1751,10 @@ var WorkerTransport = (function WorkerTransportClosure() {
             var size = width * height;
             var rgbaLength = size * 4;
             var buf = new Uint8Array(size * components);
-            var tmpCanvas = createScratchCanvas(width, height);
+            // make sure that this code is not executing in node.js.
```

Comment:
Make this comment a bit more specific, i.e., why is it bad if this runs with Node.js?